backend/organization/models/members.py

```diff
@@ -114,3 +114,68 @@ class Meta:
 
     def __str__(self):
         return "User %d member for organization %s" % (self.user.id, self.organization.name)
+
+class MembershipRequests(models.Model):
+    
+    id = models.AutoField(help_text='identifier of a join request'
+                                , primary_key=True
+                                ,verbose_name="join request Id")
+
+
+    user = models.ForeignKey(
+        User,
+        verbose_name="feedback_user",
+        help_text="Points to the user who sent the request",
+        on_delete=models.CASCADE,
+        null=False
+    )
+
+    target_membership_type = models.SmallIntegerField(choices=[(x.value,x.name) for x in MembershipTarget]
```

Comment:
Nit: I think you are applying SQL style here. In python I think this is a recommended format. 
```
target_membership_type = models.SmallIntegerField(
    choices=[(x.value, x.name) for x in MembershipTarget],
    null=False,
    verbose_name="Membership Request Target Type",
    help_text="The type of entity a user requested to join (organization, project...)"
)
```